src/display/canvas.js

```diff
@@ -38,8 +38,8 @@ const MIN_FONT_SIZE = 16;
 const MAX_FONT_SIZE = 100;
 const MAX_GROUP_SIZE = 4096;
 
-// Heuristic value used when enforcing minimum line widths.
-const MIN_WIDTH_FACTOR = 0.65;
+// Enforce line to have a minimal width of 1px when transform is applied
+const MIN_WIDTH_FACTOR = 1.000001;
```

Comment:
Also, given all the different `getSinglePixelWidth` and `MIN_WIDTH_FACTOR` usages throughout this file; are these changes generally correct/safe (as in not regressing existing tests)?

Unfortunately PR #4615 landed without a test-case included, so please also *manually* test the issues/bugs listed there to ensure that you're not regressing anything here!

Reply:
I checked all the pdf mentioned in the above PR and they all looks good.
I ran locally ref tests and the differences are minimal.
I'd say that with my eyes the rendering on firefox-linux and on acrobat-windows are very close (same screen).